docs/development.rst

```diff
@@ -0,0 +1,13 @@
+.. _development:
+
+Development
+===========
+
+You can contribute to this project forking it from github and sending pull requests.
+
```

Comment:
This would be a good place to point out that django-axes is a Jazzband project as mentioned in the [Jazzband guidelines](https://jazzband.co/about/guidelines):

```rst
This is a `Jazzband <https://jazzband.co>`_ project. By contributing you agree to abide by the `Contributor Code of Conduct <https://jazzband.co/about/conduct>`_ and follow the `guidelines <https://jazzband.co/about/guidelines>`_.
```